src/Component/RuleTable/RuleTable.tsx

```diff
@@ -193,6 +193,12 @@ export class RuleTable extends React.Component<RuleTableProps, RuleTableState> {
   }
 
   onSymbolizerClick = (record: RuleRecord, symbolizerEditorPosition: DOMRect) => {
+    const {
+      rowSelection
+    } = this.props;
+    const onChange = rowSelection.onChange;
```

Comment:
Tests are failing, because `rowSelection` can be `null` right here. Please add a check.